### ISSUE TITLE
ValueError exception apparently differs between Py2 and Py3

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#406]
We have code in mof_compiler:
```
        except ValueError as ve:
            ce = CIMError(CIM_ERR_INVALID_PARAMETER,
                          'Invalid value for property %s: %s' % \
                          (pname, ve.message))
            ce.file_line = (p.parser.file, p.lexer.lineno)
```
Works on py2 and there is a message component in ValueError.  in PY3 however it generates the following error:
```
  File "/home/kschopmeyer/pywbem/githubpywbem/pywbem/pywbem/mof_compiler.py", line 1406, in p_instanceDeclaration
    print('ve %s msg %s' %(ve, ve.message))
AttributeError: 'ValueError' object has no attribute 'message'
```
See pr #404 for fix. Fix is to drop the ve.message and do str(ve). That works for python 2 and 3

I did not find any other use of the .message attribute in pywbem code.

**Comment by andy-maier** [comment]
I agree with the proposed fix.

**Comment by andy-maier** [comment]
PR #404 fixes not only issue #402 but also this issue here. Closing it.
